### PR TITLE
[Infrastructure] Add docker and ci

### PR DESCRIPTION
**Pull request by sin-diesel**

### CODE REVIEW

File: ci-scripts/Dockerfile_install [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+FROM ubuntu:18.04
+COPY gnu-koryavov /home/gnu-koryavov
+
+RUN apt-get update
```
Comment (nerett) [inline ci-scripts/Dockerfile_install]: `apt-get` is deprecated, maybe it would be better to run `apt`?

File: scripts/install.sh [hunk 2]
```diff
@@ -8,8 +8,9 @@ read -p "Вы уверены, что хотите установить скри
 if [[ $ans == "y"* || $ans == "Y"* || $ans == "д"* || $ans == "Д"* ]]; then
 
     mkdir -p -v ~/$home_dir/KORYAVNIKS
-    cp config.conf ~/$home_dir
-    sudo cp $scripts_dir/run.sh /usr/local/bin/$home_dir
+    cp ../config.conf ~/$home_dir
+    sudo cp run.sh /usr/local/bin/$home_dir
     
     echo "Cкрипт gnu-koryavov установлен."
```
Comment (Hollbrok) [inline scripts/install.sh]: Probably there is better to use either Russian or English in all output messages.

File: ci-scripts/entrypoint_run.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+echo "Running install.sh..." &&
+chmod +x install.sh && \
+yes | ./install.sh
+
+
+semester="3"
+task="4.23"
+
+./run.sh -s $semester -n $task > output.tmp
+output_task=$(grep "Выбранный семестр:" | sed -nr 's/.*([[:digit:]]{1,4}).*/\1/p')
```
Comment (Hollbrok) [inline ci-scripts/entrypoint_run.sh]: Probably there is better to use either Russian or English in all output messages.

**Review by Hollbrok — APPROVED** [review]
Only one language should be used.

**Review by Hollbrok — APPROVED** [review]

**Pull request merged**